deployer/deployer-services/src/main/java/io/hyscale/deployer/services/model/ResourceFieldSelector.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.deployer.services.model;
+
+public enum ResourceFieldSelector {
+	POD_TEMPLATE_HASH("pod-template-hash"), CONTROLLER_REVISION_HASH("controller-revision-hash");
+
+	private String resourceFieldSelector;
+
+	ResourceFieldSelector(String resourceFieldSelector) {
+		this.resourceFieldSelector = resourceFieldSelector;
+	}
+
+	public String getResourceFieldSelector() {
+		return resourceFieldSelector;
+	}
+
```

Comment:
I don't see them being used anywhere?

Reply:
These are not selectors they should be in ResourceLabelKey if being used